sdk/trace/batch_span_processor.go

```diff
@@ -296,6 +320,10 @@ func (bsp *batchSpanProcessor) drainQueue() {
 }
 
 func (bsp *batchSpanProcessor) enqueue(sd ReadOnlySpan) {
+	bsp.enqueueBlockOnQueueFull(context.TODO(), sd, bsp.o.BlockOnQueueFull)
```

Comment:
In order to work in the ForceFlush method, it was needed to pass a context (to allow cancellation).